crates/oxc_isolated_declarations/tests/snapshots/eliminate-imports.snap

```diff
@@ -16,4 +16,8 @@ export declare function foo(this: ThisType2): void;
 export declare const bar: (this: ThisType3) => void;
 import { type InferType1, type InferType2 } from "infer";
 export type F<X extends InferType1> = X extends infer U extends InferType2 ? U : never;
+export interface VitestUtils {
+	fn: typeof fn;
+	isMockFunction: (fn: any) => fn is any;
+}
```

Comment:
this PR on it's own does not fix the bug. the `fn` in `typeof fn` is being resolved to the `fn` as the function argument. we need the new scope to fix it